Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3827,6 +3794,50 @@ void LocalFrameView::scrollToTextFragmentRange()
 
     // FIXME: <http://webkit.org/b/245262> (Scroll To Text Fragment should use DelegateMainFrameScroll)
     TemporarySelectionChange selectionChange(document, { range }, { TemporarySelectionOption::RevealSelection, TemporarySelectionOption::RevealSelectionBounds, TemporarySelectionOption::UserTriggered, TemporarySelectionOption::ForceCenterScroll });
+
+    if (m_delayedTextFragmentIndicatorTimer.isActive())
+        return;
+
+    auto textIndicator = TextIndicator::createWithRange(range, { TextIndicatorOption::DoNotClipToVisibleRect }, WebCore::TextIndicatorPresentationTransition::Bounce);
+
+    auto* page = m_frame->page();
+
+    if (!page)
+        return;
+
+    if (textIndicator) {
+        RefPtr localMainFrame = page->localMainFrame();
+        if (!localMainFrame)
+            return;
+
+        auto textRects = RenderObject::absoluteTextRects(range);
+
+        constexpr auto hitType = OptionSet { HitTestRequest::Type::ReadOnly, HitTestRequest::Type::Active, HitTestRequest::Type::AllowVisibleChildFrameContentOnly };
```

Comment:
You can skip `auto` if you use direct initialization
```suggestion
        static constexpr OptionSet hitType { HitTestRequest::Type::ReadOnly, HitTestRequest::Type::Active, HitTestRequest::Type::AllowVisibleChildFrameContentOnly };
```

Also, optional, but consider making it `static`.